staging/src/github.com/kubeedge/mapper-framework/_template/mapper/data/stream/handler_nostream.go

```diff
@@ -1,3 +1,5 @@
+//go:build nostream
+
```

Comment:
There is no need to add cross-compilation conditions, this part will be processed in the mapper-framework generation script